scripts/joint_limits_observer.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+import rospy
+import sys
+from giskardpy import urdf_object
+from sensor_msgs.msg import JointState
+
+# Visualizes the given joint states w.r.t. their limits
+# rosrun giskardpy joint_limit_observer <joint_state_topic>
+
+warning_threshold = 0.1
+critical_threshold = 0.02
+scale_steps = 20
+
+
+class OutColor:
+    WARNING = '\033[93m'
+    FAIL = '\033[91m'
+    NORM = '\033[0m'
+
+
+class JSObserver(rospy.Subscriber):
+    def __init__(self, robot, topic, message_type):
+        self.joint_limits = robot.get_all_joint_limits()
+        super(JSObserver, self).__init__(topic, message_type, self.js_cb)
+
+    def js_cb(self, msg):
+        output = ""
```

Comment:
To make it consistent with my other code and to make a possible transition to python 3 easier, can you please use single quotes and unicode for strings? e.g. `u'muh'`